hiddenregions.xml

```diff
@@ -0,0 +1,79 @@
+<!--
+	hiddenregions.xml
+	made by oatmealine
+
+	lets you add hidden regions to a player as a function
+
+	hide {beat, duration, [column] [plr]}
+	```lua
+	hide
+	{0, 2, plr = 1, column = {0, 1, 2}}
+	{2, 2, plr = 2, column = {0, 1, 2}}
+	```
+-->
+
+<Layer Type = "Quad" LoadCommand = "%xero(function(self)
+	local hiddenregions = {}
+	
+	local function contains(table, element)
+		for _, value in pairs(table) do
+			if value == element then
+				return true
+			end
+		end
+		return false
+	end
+
+	local function throw(msg)
+		local _, err = pcall(error, msg, 4)
+		SCREENMAN:SystemMessage(err)
+	end
+
+	function hide(table)
+		if type(table) == 'table' then
+			local start = table[1]
+			local dur = table[2]
+
+			local players = table.plr or plr or {1, 2, 3, 4}
```

Comment:
read rawget(xero, 'plr') instead of 'plr', so that there aren't issues if the theme sets a 'plr' variable

Reply:
you did this just now